scripts/popup.js

```diff
@@ -13,7 +15,7 @@ async function setSettings(newSettings) {
 function showToast() {
     const toast = document.getElementById("snackbar");
     toast.classList.add("show");
-    setTimeout(() => toast.classList.remove("show"), 3000);
+    setTimeout(() => toast.classList.remove("show"), 2900);
```

Comment:
any reason to show it 100 ms less? 🤔

Reply:
The old one was glitching for me this did the trick.